File: seed/csharp-sdk/exhaustive/no-root-namespace-for-core-classes/Snippets/Example25.cs
using SeedExhaustive;
using SeedExhaustive.Core;

public partial class Examples
{
    public async Task Example25() {
        var client = new SeedExhaustiveClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Endpoints.Object.GetAndReturnMapOfDocumentedUnknownTypeAsync(
            new Dictionary<string, object>(){
                ["string"] = new Dictionary<string, object>()
                {
                    ["key"] = "value",
                }
                ,
            }
        );
    }

}

File: seed/swift-sdk/exhaustive/Tests/Snippets/Example25.swift
import Foundation
import Exhaustive

enum Example25 {
    static func snippet() async throws {
        let client = ExhaustiveClient(
            baseURL: "https://api.fern.com",
            token: "<token>"
        )

        _ = try await client.endpoints.object.getAndReturnMapOfDocumentedUnknownType(request: [
            "string": .object([
                "key": .string("value")
            ])
        ])
    }
}
